### ISSUE TITLE
Driver Uses Variably-Sized Stack Variables

### ISSUE THREAD

**Issue by CN-YongJun** [Sensirion/embedded-sgp#67]
![image](https://user-images.githubusercontent.com/18485997/64395872-5286a500-d08e-11e9-9b29-2b5a0d3be1cc.png)

**Comment by CN-YongJun** [comment]
expression must have a constant value.

**Comment by abrauchli** [comment]
yes, some compilers have an issue with that and we will fix it. For a quick fix you can assume that a buffer of three words will always work.

```diff
@@ -90,7 +87,7 @@ static struct sgpc3_data {
 static void unpack_signals(const struct sgp_profile *profile) {
     int16_t i, j;
     uint16_t data_words = profile->num_words;
-    uint16_t word_buf[data_words];
+    uint16_t word_buf[3];
 
     /* copy buffer */
     for (i = 0; i < data_words; i++)
```

**Comment by GetPastTheMonkey** [comment]
This should be fixed now (see commit c955989b0bb1c6d3164456fad15e1512c62f5862).

@CN-YongJun Could you please check if it works now for you?
